wagtail/wagtailembeds/oembed_providers.py

```diff
@@ -301,6 +301,9 @@
     "http://issuu.com/oembed": [
         "^http(?:s)?://(?:www\\.)?issuu\\.com/[^#?/]+/docs/.+$"
     ],
+    "https://www.tumblr.com/oembed/1.0": [
+        "^http(?:s)?://.+.tumblr.com/post/.+$",
```

Comment:
A few improvements to the regexp:
- You must escape the explicit dots in the domain name.
- You can remove the non-capturing group for 's' of 'https'. It could of course be removed on all other oembed regexps.
- Maybe a non-greedy expression for the subdomain would be better.

So, something like:
```python
r'^https?://.+?\.tumblr\.com/post/.+$',
```
or
```python
r'^https?://[^\.]+\.tumblr\.com/post/.+$',
```